docs/requirements.md

```diff
@@ -2,78 +2,48 @@
 
 Running Codacy on a Kubernetes cluster requires the following:
 
--   A Kubernetes 1.14.\* or 1.15.\* cluster provisioned with the [required resources](#cluster-resource-requirements)
--   The [NGINX Ingress Controller](https://github.com/helm/charts/tree/master/stable/nginx-ingress) for Kubernetes
+-   A Kubernetes 1.14.\* or 1.15.\* cluster provisioned with the [required resources](#hardware-requirements)
+-   The [NGINX Ingress Controller](https://github.com/helm/charts/tree/master/stable/nginx-ingress) must be setup in your cluster
 -   A [PostgreSQL server](#postgresql-server-setup) accessible from the Kubernetes cluster
 
-## Cluster resource requirements
+## Hardware Requirements
 
-To have an enjoyable experience with Codacy, you should have the
-following calculations in mind when allocating resources for the
-installation and defining the number of concurrent analysis.
+A good way of understanding how resources are used in Codacy is by understanding how the resources are allocated.
 
-Without accounting for analysis ([next section](#analysis)),
-you should need at least the following resources:
+!["High Level Architecture"](<./images/High Level Architecture - Analysis II.svg> "High Level Architecture")
 
-```text
-CPU: 7 CPU
-Memory: 40 GB
-```
+You can look at Codacy separately as the "Platform" and the "Analysis" cluster. The Platform contains the UI and other
+components important to treat and show results. The Analysis is the swarm of workers that run several linters for
+your project.
```

Comment:
I would place this before the architecture diagram as it helps the reader understand what they should be looking at in the diagram.